### PR TITLE
use format_placeholder for wireless_info

### PR DESCRIPTION
**Pull request by Stunkymonkey**
related to #384

### CODE REVIEW

**Comment by Airblader** [comment]
@Stunkymonkey Can you look into the failed test? Thanks!

File: src/print_wireless_info.c [hunk 1]
```diff
@@ -517,7 +519,12 @@ void print_wireless_info(yajl_gen json_gen, char *buffer, const char *interface,
         if (ipv6_address == NULL) {
             START_COLOR("color_bad");
             outwalk += sprintf(outwalk, "%s", format_down);
-            goto out;
```
Comment (Airblader) [inline src/print_wireless_info.c:520]: Why was this necessary? This now just duplicates what happens in `out` except for the placeholders, right?

Comment (Stunkymonkey) [reply src/print_wireless_info.c:520]: sure, but otherwise i could not distinguish in the end between "interface down/up". If the interface is up i have to free the buffer after the printing.

Comment (Stunkymonkey) [reply src/print_wireless_info.c:520]: and having `goto` labels is not always lucid

Comment (Airblader) [reply src/print_wireless_info.c:520]: > If the interface is up i have to free the buffer after the printing.

Right, point taken.

> and having goto labels is not always lucid

There are several ways to avoid duplicating code; macros, functions, goto. ;-) Since in this case it's actually (slightly) different and not too much I think it's OK.
